src/platform/elements/value/Value.ts

```diff
@@ -131,11 +100,16 @@ export class NovoValueElement implements OnInit, OnChanges {
         return this.theme === NOVO_VALUE_THEME.MOBILE;
     }
 
-    public get iconClass(): string {
-        if (this.meta && this.meta.icon) {
-            return `bhi-${this.meta.icon} actions`;
+    iconClass(icon): string {
+        let iconClass = '';
+        if (icon && icon.iconCls) {
+            iconClass = `bhi-${icon.iconCls} actions`;
+            if (icon.onIconClick) {
+                iconClass = `${iconClass} clickable`
```

Comment:
can you add styles for the actions class to hide the cursor (pointer) on hover and only show the pointer cursor when clickable is active?

Reply:
yes will do